features/exploredash/src/main/java/org/dash/wallet/features/exploredash/repository/ExploreRepository.kt

```diff
@@ -77,7 +77,7 @@ class GCExploreDatabase @Inject constructor(
             return if (isTestDB) {
                 "explore/explore-testnet.db"
             } else {
-                "explore/explore.db"
+                "explore/explore-devnet.db" // this is temporary, should have means of selecting dev db
```

Comment:
for now we will use the `devnet` db during development.

However, when we move to production, perhaps we should call it `explore-2.db`

Then the Cloud Storage will have:
1. `explore.db` - older Dash Wallet apps will see this v1 file and sync with it.  
2. `explore-2.db` - newer apps will see this v2 file and sync with it.

This way old apps don't download v2 files which will be considered invalid.  Not sure if a v2 db will end up clearing out all merchants with a destructive fallback or if the app will try to sync with the updated db many, many times.

Reply:
Since this is for preloaded databases, then we don't need to rename it.